src/site/stages/build/process-cms-exports/schemas/raw/taxonomy_term-health_care_service_taxonomy.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable camelcase */
+
+module.exports = {
+  type: 'object',
+  properties: {
+    name: { $ref: 'GenericNestedString' },
+    description: {
+      oneOf: [
+        { $ref: 'GenericNestedString' },
+        {
+          type: 'array',
+          items: { type: 'object', properties: { value: { type: 'null' } } },
+        },
```

Comment:
If we run into this again, it might make sense to make a `NullableNestedString` or something.